ui/app/AppLayouts/Communities/panels/PermissionsSettingsPanel.qml

```diff
@@ -13,6 +13,8 @@ import shared.popups 1.0
 StackView {
     id: root
 
+    property var communityTokensStore
```

Comment:
Stores should be always strongly typed.

Reply:
I agree! It's just part of the commit to remove. It adds some testing logic for you to easily visualize all toasts introduced in here.